src/web/layouts/Standard.stories.tsx

```diff
@@ -9,7 +9,7 @@ import { Analysis } from '@root/fixtures/articles/Analysis';
 import { Feature } from '@root/fixtures/articles/Feature';
 import { GuardianView } from '@root/fixtures/articles/GuardianView';
 import { Interview } from '@root/fixtures/articles/Interview';
-import { Quiz } from '@root/fixtures/articles/Quiz';
+import { KnowledgeQuiz } from '@root/fixtures/articles/KnowledgeQuiz';
```

Comment:
👍  for the story but it might make more sense here to keep the name `Quiz` here as that is the `designType` value and the rest of these articles are named based on design type.